adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -743,7 +751,7 @@ def play_tone(self, frequency, duration, waveform=SINE_WAVE):
         time.sleep(duration)
         self.stop_tone()
 
-    def start_tone(self, frequency, waveform=SINE_WAVE):
+    def start_tone(self, frequency: int, waveform: int = SINE_WAVE) -> None:
```

Comment:
Same thing, mind updating the type annotation in the docstring?

Reply:
It looks like this type annotation wasn't corrected for the docstring, would you mind updating it to much your annotation?

Reply:
The `waveform` parameter type in the docstring should be `int` instead of `str`.

Reply:
fixed